konig-core/src/main/java/io/konig/core/vocab/XOWL.java

```diff
@@ -0,0 +1,32 @@
+package io.konig.core.vocab;
+
+/*
+ * #%L
+ * Konig Core
+ * %%
+ * Copyright (C) 2015 - 2018 Gregory McFall
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import org.openrdf.model.URI;
+import org.openrdf.model.impl.URIImpl;
+
+public class XOWL {
+	public static final URI TERMSTATUS = new URIImpl("http://schema.pearson.com/ns/xowl/termstatus");
```

Comment:
We are using the camelCase convention for property names.

Please replace `termstatus` with `termStatus` so that the fully-qualified URI is
```
http://schema.pearson.com/ns/xowl/termStatus
```
  
Also, please replace the static field name `TERMSTATUS` with `termStatus`.